bin/run_unittests.sh

```diff
@@ -107,6 +107,8 @@ function run_libclipper_tests {
 function run_management_tests {
   echo -e "\nRunning management tests\n\n"
   ./src/management/managementtests --redis_port $REDIS_PORT
+  cd $DIR
+  python ../clipper_admin/test/clipper_manager_test.py all
```

Comment:
Made the `clipper_manager` tests a separate function (`clipper_admin_tests`) that can be executed separately with option `-c/--clipperadmin`. Inserted a sleep between `management_tests` and `clipper_admin_tests`.